desktop/apps/auction2/components/basic_checkbox/index.js

```diff
@@ -0,0 +1,24 @@
+import { default as React, PropTypes } from 'react';
+
+export default function BasicCheckbox({ item, onClickAction, checked, onClick }) {
+  return (
+    <div className='auction2-basic-checkbox'>
+      <div className='artsy-checkbox' onClick={() => onClick(item.id)}>
+        <div className='artsy-checkbox--checkbox'>
+          <input
+            type='checkbox'
+            name='artistAggregation'
+            value={item.id}
+            checked={checked}
+            onChange={() => {}} // necessary to get around warning
```

Comment:
@sweir27 - Have you seen this? https://facebook.github.io/react/docs/uncontrolled-components.html#default-values